File: external/cpp-impl/src/hgoe/edges/related_to_edge.cpp
//
// Created by jldevezas on 3/8/18.
//

#include <hgoe/edges/related_to_edge.h>

BOOST_CLASS_EXPORT_IMPLEMENT(RelatedToEdge)

RelatedToEdge::RelatedToEdge() {

}

RelatedToEdge::RelatedToEdge(NodeSet nodes) : Edge(nodes) {

}

Edge::Label RelatedToEdge::label() const {
    return Label::RELATED_TO;
}

File: external/java-impl/src/main/java/armyant/hgoe/edges/RelatedToEdge.java
package armyant.hgoe.edges;

/**
 * Created by jldevezas on 2017-10-25.
 */
public class RelatedToEdge extends Edge {
    private String relation;

    public RelatedToEdge() {
        this("related_to");
    }

    public RelatedToEdge(String relation) {
        super();
        this.relation = relation;
    }
}
